sandbox/app.js

```diff
@@ -123,6 +124,22 @@ function on_error(err, req, res, next) {
 }
 
 const handlers = require('./handlers');
+app.use(
+  cors({
+    origin: 'https://digital.nhs.uk',
```

Comment:
This is hardcoded to the actual location the sandbox runs in. The other option here is to wildcard and allow all origins.